uber/decorators.py

```diff
@@ -30,6 +30,15 @@ def with_check(*args, **kwargs):
     return with_check
 
 
+def redirect_if_at_con_to_kiosk(func):
+    @wraps(func)
+    def with_check(*args, **kwargs):
+        if c.AT_THE_CON:
```

Comment:
Minor improvement which I'll probably just go ahead and make now: this should also check for the existence of the URL, so we can 'turn off' this functionality by simply blanking out the kiosk redirect URL